examples/simple_hpc_pbs_zfs/pipeline.yml

```diff
@@ -0,0 +1,45 @@
+pr: none
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - '*'
+  paths:
+    include:
+      - examples/simple_hpc_pbs_zfs/*
+      - pyazhpc/*
```

Comment:
Please remove that line as we don't want this example to be triggered for every python code change in azhpc

Reply:
fixed